src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,28 @@
+import classNames from 'classnames';
+
+type Props = {
+  showError: boolean,
+  error: string,
+  onErrorClose: () => void,
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  showError,
+  error,
+  onErrorClose
+}) => (
+  <div
+    className={classNames(
+      'notification is-danger is-light has-text-weight-normal',
+      { hidden: !showError },
+    )}
+  >
+    <button
+      aria-label="Close"
+      type="button"
+      className="delete"
+      onClick={onErrorClose}
+    />
+    {error}
+  </div>
+)
\ No newline at end of file
```

Comment:
Add a newline at the end of the file. It's good practice to have a newline at the end of the file to avoid issues when concatenating files.
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file